Misc/NEWS.d/next/Library/2024-01-28-19-40-40.gh-issue-114678.kYKcJw.rst

```diff
@@ -0,0 +1,2 @@
+Fix incorrect deprecation warning for 'N' specifier in
+:class:`~decimal.Decimal` format. Based on patch by Stefan Krah.
```

Comment:
I think this would read better as something like "Ensure that deprecation warning for 'N' specifier in :class:`~decimal.Decimal` format is not raised for cases where 'N' appears in other places in the format specifier."

I initially read read it as saying that 'N' was being un-deprecated.

Reply:
Thank you for suggestion. Done.